### PR TITLE
Add Java workers

### PR DESCRIPTION
**Pull request by mliszcz**
PR contents:
* Java based workers (#64)
* Java worker for pushevent (#14)
* Cpp worker for pushevent (#14)
* README update
* Workers are installed in docker as well

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +75,14 @@ For client side (benchmark runners):
     # or make install INSTALL_DIR=~/.local/bin
     ```
 
+1. Install Java workers:
+    ```bash
+    cd javaclient
+    for d in tg-benchmark-client-*; do cd $d && mvn package; cd ..; done
```
Comment (piogor) [inline README.md]: @mliszcz, these installation lines are quite a bash script. What do you think about adding a javaclient-install.sh to javaclient folder?

**Review by piogor — CHANGES_REQUESTED** [review]
@mliszcz, thanks! Since you propose for java workers/client a clever template-based installation procedure (which I like a much), I suggest to move it to a script then update the README.md accordingly.

File: README.md (hunk 1 above)
Comment (mliszcz) [reply README.md]: Fixed in 985e089802d64ae02ec1e167acf4704af96649c6

**Comment by mliszcz** [comment]
@piogor thanks for the review. Please check if it's ok now.

**Review by piogor — APPROVED** [review]
Thanks, @mliszcz ! Sorry for not merging for so long ;-)

**Pull request merged**